### PR TITLE
todomvc web-components, change title, id & completed attributes on item component

### PR DESCRIPTION
**Pull request by flashdesignory**
issue https://github.com/WebKit/Speedometer/issues/313

changed this.id => this["item-id"], this.title => this["item-title"] and this.completed => this["item-completed"].
If the brackets are bothersome, I could change it to this.itemid, this.itemtitle, ect... just personal preference to use the above. 

chrome before:
before_chrome

chrome after:
after_chrome

firefox before:
before_firefox

firefox after:
after_firefox

safari before:
before_safari

safari after:
after_safari

edge before:
before_edge

edge after:
after_edge

@kara

### CODE REVIEW

**Comment by julienw** [comment]
If that's OK with you, I'd rather have properties without the dashes so that we don't need the brackets indeed. Please use camelCase (itemId, itemTitle, etc).

**Review by julienw — APPROVED** [review]

File: resources/todomvc/vanilla-examples/javascript-web-components/dist/components/todo-list/todo-list.component.js [hunk 1]
```diff
@@ -33,8 +33,12 @@ class TodoList extends HTMLElement {
     }
 
     addItem(entry) {
+        const { id, title, completed } = entry;
         const element = new TodoItem();
-        Object.keys(entry).forEach((key) => element.setAttribute(key, entry[key]));
+
+        element.setAttribute("itemid", id);
+        element.setAttribute("itemtitle", title);
+        element.setAttribute("itemcompleted", completed);
```
Comment (camillobruni) [inline resources/todomvc/vanilla-examples/javascript-web-components/dist/components/todo-list/todo-list.component.js:41]: Can't you set the properties directly here?
The `connectedCallback` should take care of updating them, no?

Comment (flashdesignory) [reply resources/todomvc/vanilla-examples/javascript-web-components/dist/components/todo-list/todo-list.component.js:41]: I do want also to have the id, title, completed attributes on the custom element. I use these attributes to check on status of the items. 

with the attributes: 
```
<todo-item dir="ltr" itemid="jqT4S4j6I6qSuy3SUUgBo" itemtitle="hi" itemcompleted="false" data-priority="4" style="display: block;"></todo-item>
```

without: 
```
<todo-item dir="ltr" data-priority="4" style="display: block;"></todo-item>
```

Comment (camillobruni) [reply resources/todomvc/vanilla-examples/javascript-web-components/dist/components/todo-list/todo-list.component.js:41]: ah right, make sense.

**Review by rniwa — APPROVED** [review]

**Pull request merged**